src/roslyn/src/Compilers/Server/VBCSCompiler/CompletionData.cs

```diff
@@ -35,9 +35,9 @@ internal readonly struct CompletionData
     {
         internal CompletionReason Reason { get; }
         internal TimeSpan? NewKeepAlive { get; }
-        internal bool ShutdownRequest { get; }
+        internal string? ShutdownRequest { get; }
```

Comment:
Would it make sense to update the property name? `ShutdownRequestedBy` or similar?